documentation/general/dotnet-run-file.md

```diff
@@ -172,56 +172,54 @@ App/Program2/bin/
 App/Program2/obj/
 ```
 
-## Package references
+## Directives for project metadata
 
-It is possible to specify NuGet package references via the `#package` directive.
+It is possible to specify some project metadata via [ignored C# directives][ignored-directives].
+Directives `sdk`, `package`, and `property` are translated into `<Project Sdk="...">`, `<PackageReference>`, and `<Property>` project elements, respectively.
+Other directives result in a warning, reserving them for future use.
 
 ```cs
-#package Newtonsoft.Json 13.0.1
+#:sdk      Microsoft.NET.Sdk.Web
+#:property TargetFramework=net11.0
+#:property LangVersion=preview
+#:package  System.CommandLine=2.0.0-*
 ```
 
-The C# language needs to be updated to ignore these directives (instead of failing the compilation).
-See [the corresponding language proposal][pound].
+The value must be separated from the name of the directive by white space and any leading and trailing white space is not considered part of the value.
+The value of `#:sdk` is injected into `<Project Sdk="{0}">` as is.
+The value of `#:property` is split by the first `=` and injected as `<{0}>{1}</{0}>` in a `<PropertyGroup>`.
```

Comment:
Can you pass multiple like:
#property foo=fooV;bar=barV
etc.?

Reply:
You can pass multiple like

```cs
#:property foo=fooV
#:property bar=barV
```

Reply:
That's reasonable. I'm a bit split as to whether I think it's best to leave it like that or also permit the #property foo=fooV;bar=barV syntax. On the one hand, MSBuild (from the command line) supports either; on the other hand, I don't think that enables anything new, and this syntax is more similar to C# code, which feels appropriate. I think I'm happy with this.